ruby.rb

```diff
@@ -4,42 +4,71 @@
 # 1. Use TWO different Ruby methods to return a new array with all the numbers multiplied by 3. Expected output: [3, 6, 18, 27, 9, 63]
 
 myArray = [1, 2, 6, 9, 3, 21]
+def array
+  num = 1
+  array.each do |i|
+    num *= i
+  end
+  p num
+end
 
+myArray = [1, 2, 6, 9, 3, 21]
 
+def array
+  return myArray.inject(&:*)
+end
 
 # 2. Create a method that takes in a sentence and returns a new sentence with the first letter of each word capitalized. Expected output = "Hello There, How Are You?"
 
 sentence = "hello there, how are you?"
-
+def sen(sentence)
+  sentence.upcase
+end
+  
 
```

Comment:
Remember to make your argument a different name than the variable you are passing to your method. You should always call your method to make sure you are getting the expected outcome. This is one way to do it, although there are others.
def make_cap string
  new_stuff = string.split.map do
    |word| word.capitalize
  end
  new_stuff.join(" ")
end
p make_cap sentence